tests/utils/predatorApiHelper.js

```diff
@@ -45,6 +45,26 @@ module.exports.createJob = async (testId) => {
     }
 };
 
+module.exports.getAggregatedReports = async (testId, reportId) => {
+    const options = {
+        url: PREDATOR_URL + `/tests/${testId}/reports/${reportId}/aggregate`,
+        method: 'GET',
+        json: true,
+        headers: {
+            'x-zooz-request-id': 'mickey'
```

Comment:
maybe change these headers to `x-runner-id` so that way predator audits the runners that interact with it? or do we want to keep the x-zooz-request-id?

Reply:
done